ansible/advanced.md

```diff
@@ -84,6 +94,20 @@ meanings for different followers.
 * ER-301 can be sent gate and CV commands, with the 4 tracks
   corresponding to the first 4 `SC` channels.
 
+* Disting Ex, left to right: allocate notes/gates polyphonically, map each
+  track to one output, convert CV to MIDI via the MIDI breakout on the
+  Disting EX, all channels sent via channel 1, all channels send
+  via channels 1-4. In MIDI modes data is passed through the Disting,
+  it cannot be used in single algorithm mode, but will function
+  in dual algorithm mode.
+
+* W/ Syn, left to right: allocate notes polyphonically, map each track
```

Comment:
@csboling , this is all correct to the most recent changes, yeah?

Reply:
Yes, this seems right. The Ansible code that implements this is around [here](https://github.com/monome/ansible/blob/4bb07a26165df904628ce415cc6a785ee62b66a1/src/ansible_ii_leader.c#L463). I don't know much about the Disting functionality since I'm not very familiar with the many capabilities of that module.